db/seeders/20210107223757-demo-spotcollection-seed.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const { User, Collection } = require('../models');
+
+
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+
+    const demoUser = await User.findOne({ where: { username: 'demouser' } });
+
+    const demoCollections = await Collection.findAll({
+        where: {userId: demoUser.id }
+      });
+
+
+
+
+ /*
+      Add altering commands here.
+      Return a promise to correctly handle asynchronicity.
+
+      Example:
+      */
+      return queryInterface.bulkInsert('SpotCollections', [
+        { collectionId: demoCollections[0].id, spotId: 1, createdAt: new Date(), updatedAt: new Date()},
+        { collectionId: demoCollections[0].id, spotId: 2, createdAt: new Date(), updatedAt: new Date()},
+        { collectionId: demoCollections[0].id, spotId: 3, createdAt: new Date(), updatedAt: new Date()},
+        { collectionId: demoCollections[0].id, spotId: 4, createdAt: new Date(), updatedAt: new Date()},
+        { collectionId: demoCollections[0].id, spotId: 5, createdAt: new Date(), updatedAt: new Date()},
+        { collectionId: demoCollections[1].id, spotId: 6, createdAt: new Date(), updatedAt: new Date()},
+        { collectionId: demoCollections[1].id, spotId: 7, createdAt: new Date(), updatedAt: new Date()},
+        { collectionId: demoCollections[1].id, spotId: 8, createdAt: new Date(), updatedAt: new Date()},
+        { collectionId: demoCollections[1].id, spotId: 9, createdAt: new Date(), updatedAt: new Date()},
+        { collectionId: demoCollections[1].id, spotId: 10, createdAt: new Date(), updatedAt: new Date()},
+
+      ], {});
+  },
+
+  down: (queryInterface, Sequelize) => {
+    /*
+      Add reverting commands here.
+      Return a promise to correctly handle asynchronicity.
+
+      Example:
+      */
+    return queryInterface.bulkDelete('SpotCollections', null, {});
```

Comment:
In case it may be helpful, the `null` in the `bulkDelete` method is a where clause to select specific SpotCollections based on some criteria, like userId or collectionId. It could make undoing seeds more graceful.

Reply:
Great point Ben. If you'd like to specifically delete the SpotCollections for the demo user here you could replace the `null` with a clause to select the demo user's collections. If you don't want to do that that's fine too.

Reply:
i've had a look at how to implement this but I wasn't able to find in the docs how to handle the where statement in this method.  i'll leave it for now but we can come back to it if we need to.